src/main/java/net/revelc/code/impsort/UnknownLineEndingException.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.revelc.code.impsort;
+
+import java.io.IOException;
+import java.nio.file.Path;
+
+/**
+ * Signals that the file denoted by this path has an unknown line ending.
+ *
+ * <p>
+ * This exception will be thrown by the {@link ImpSort#parseFile} when it encounters a file with an
+ * unknown line ending.
+ * </p>
+ */
+public class UnknownLineEndingException extends IOException {
+  private static final long serialVersionUID = 4417291768648259852L;
```

Comment:
```suggestion
 */
public class UnknownLineEndingException extends IOException {
  private static final long serialVersionUID = 1L;
```

Reply:
OK. I believe it's the one that it generates based on the class signature and not random, but I will change to `1L`.